package.json

```diff
@@ -20,7 +19,7 @@
     "gulp-sass": "^2.3.2",
     "gulp-sourcemaps": "^1.12.0",
     "gulp-uglify": "^1.5.4",
-    "handlebars": "^3.0.3",
+    "handlebars": "^4.0.11",
```

Comment:
Do we even use handlebars in this project anywhere..?

Reply:
Initially I tried removing it and thought it worked, but it's used by a mailer that doesn't declare it as a dependency :(